src/commitlint/index.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { Tree, VirtualTree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree
+} from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+
+const collectionPath = path.join(__dirname, '../collection.json');
+
+describe('@co-it/schematics:commitlint', () => {
+  let runner: SchematicTestRunner;
+  let treeBefore: Tree;
+
+  beforeEach(() => {
+    runner = new SchematicTestRunner('commitlint', collectionPath);
+    treeBefore = new UnitTestTree(new VirtualTree());
```

Comment:
Better use `new EmptyTree` instead of `VirtualTree`.

Reply:
I used `Tree.empty()` now. Is there a difference?

Reply:
It is fine as well.
It should be the same.